imagebuildah/executor.go

```diff
@@ -47,6 +47,7 @@ var builtinAllowedBuildArgs = map[string]bool{
 	"ftp_proxy":   true,
 	"NO_PROXY":    true,
 	"no_proxy":    true,
+	"FROM":        true,
```

Comment:
Where are these documented?  Doesn't this new one need documentation in a man page.  Currently this seems like hidden magic?

Reply:
https://docs.docker.com/engine/reference/commandline/build/

Could you craft a section in the buildah man page from the above to document these as well as your new FROM arg.